utils/python/CIME/case_test.py

```diff
@@ -12,20 +13,7 @@ def case_test(case, testname=None):
     expect(testname is not None, "testname argument not resolved")
     logging.warn("Running test for %s" % testname)
 
-    try:
-        test = find_system_test(testname, case)(case)
-        success = test.run()
-
-        test.report()
```

Comment:
baseline actions and reporting are now part of the core run framework in system_test_common.